applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_integration_schemes.cpp

```diff
@@ -24,14 +26,33 @@ using namespace Kratos;
 
 namespace
 {
-
 std::unique_ptr<IntegrationScheme> MakeLobattoIntegrationScheme(std::size_t NumberOfPoints)
 {
     return std::make_unique<LobattoIntegrationScheme>(NumberOfPoints);
 }
 
 std::vector<std::size_t> SupportedNumbersOfPointsForLobattoIntegration() { return {2, 3}; }
 
+std::unique_ptr<IntegrationScheme> MakeLumpedIntegrationScheme(std::size_t NumberOfPoints)
+{
+    return std::make_unique<LumpedIntegrationScheme>(NumberOfPoints);
+}
+
+std::vector<std::size_t> SupportedNumbersOfPointsForTriangleLumpedIntegration() { return {3, 6}; }
+
+std::vector<std::size_t> SupportedNumbersOfPointsForQuadrilateralLumpedIntegration()
+{
+    return {4, 8};
+}
+
+std::vector<std::size_t> SupportedNumbersOfPointsForLumpedIntegration()
+{
+    auto result       = SupportedNumbersOfPointsForTriangleLumpedIntegration();
+    auto sup_quad_num = SupportedNumbersOfPointsForQuadrilateralLumpedIntegration();
+    std::move(sup_quad_num.begin(), sup_quad_num.end(), std::back_inserter(result));
```

Comment:
I have never seen `std::move` being used in this way, so I've learned something new today. :-) I have looked up the reference documentation on cppreference.com, and I think I would use it only for objects that are either expensive to copy or move-only. Since neither of these apply here, I would prefer a more common approach, which is to copy the elements from one vector to the other. In other words, I would probably either use the `insert` member:
```suggestion
    const auto sup_quad_num = SupportedNumbersOfPointsForQuadrilateralLumpedIntegration();
    result.insert(result.end(), sup_quad_num.begin(), sup_quad_num.end());
```
or copy `sup_quad_num` to `result`:
```suggestion
    const auto sup_quad_num = SupportedNumbersOfPointsForQuadrilateralLumpedIntegration();
    std::copy(sup_quad_num.begin(), sup_quad_num.end(), std::back_inserter(result));
```

Reply:
done